src/client/ns_turn_msg.c

```diff
@@ -2432,7 +2434,8 @@ static int encode_oauth_token_gcm(const u08bits *server_name, encoded_oauth_toke
 		ns_bcopy(dtoken->enc_block.mac_key,orig_field+len,dtoken->enc_block.key_length);
 		len += dtoken->enc_block.key_length;
 
-		*((uint64_t*)(orig_field+len)) = nswap64(dtoken->enc_block.timestamp);
+		uint64_t ts = nswap64(dtoken->enc_block.timestamp);
+		ns_bcopy( &ts, (orig_field+len), 8);
```

Comment:
Use `sizeof(ts)` here and in the next line?

Reply:
I will fix it.